File: Domain/Entities/ET/EvaluateGroup.cs
﻿using System.Collections.Generic;

namespace Domain.Entities.ET;

public class EvaluateGroup : EntityBase
{
    public string EvaluateGroupCode { get; set; }
    public string EvaluateGroupNameTh { get; set;}
    public string EvaluateGroupNameEn { get; set; }
    public int SequeneId { get; set; }
    public int? TotalPoint {  get; set; }
    public bool? Active { get; set; }
    public string RoleCode { get; set; }
    public ICollection<EvaluateDetail> EvaluateDetails { get; set; }
}

File: Web/ClientApp/src/app/models/et/evaluateGroup.ts
import { EntityBase } from "../entityBase";
import { EvaluateDetail } from "./evaluateDetail";

export class EvaluateGroup extends EntityBase {
    evaluateGroupCode: string;
    evaluateGroupNameTh: string;
    evaluateGroupNameEn: string;
    roleCode: string;
    sequeneId: number;
    totalPoint: number;
    lanuageCode: string;
    active: boolean;
    evaluateDetails:  EvaluateDetail[]

    constructor() {
        super();
        this.evaluateDetails = []
    }
}